src/main/java/mekanism/client/gui/GuiLogisticalSorter.java

```diff
@@ -511,11 +522,15 @@ protected void drawGuiContainerForegroundLayer(int mouseX, int mouseY) {
         }
 
         if (xAxis >= 12 && xAxis <= 26 && yAxis >= 110 && yAxis <= 124) {
-            drawHoveringText(LangUtils.localize("gui.autoEject"), xAxis, yAxis);
+            drawHoveringText(LangUtils.localize("gui.logisticalSorter.autoEject.tooltip"), xAxis, yAxis);
```

Comment:
I think the tooltips (this one and the other two added) should wrap the text rather than just straight up be drawn. This is because some of the descriptions are somewhat long so if you do not have Minecraft in full screen it can run off the edge slightly. (Should be manageable automatically similar to how our item tooltips have their description.)

Reply:
Good point, will look into it 👍

Reply:
Please also ensure any new language keys are prefixed with `mekanism.`